apps/core/lib/core/schema/repository.ex

```diff
@@ -54,6 +54,13 @@ defmodule Core.Schema.Repository do
       field :url,  :string
     end
 
+    embeds_one :community, Community, on_replace: :update do
+      field :discord, :string
+      field :slack,   :string
+      field :twitter, :string
+      field :videos,  {:array, :string}
```

Comment:
one thing that might be a good idea here is to make this a structure with video type (since embedding youtube vs vimeo vs standard video urls is a bit different).  If there's a prebaked video component that can discover this from url alone, it's unnecessary though

Reply:
https://github.com/CookPete/react-player seems to be perfect for this. WDYT?

Reply:
Yeah agreed